tests/autorag/nodes/passagereranker/test_monot5.py

```diff
@@ -0,0 +1,7 @@
+from autorag.nodes.passagereranker import monot5
+
+from tests.autorag.nodes.passagereranker import test_base_reranker
+
+
+def test_monot5():
+    test_base_reranker.rerank_test(monot5)
```

Comment:
use [modal](https://modal.com/) for this test.
Because it is taking so much time at CI/CD because it can't use CUDA.

Reply:
i'll make apply modal issue. coming soon at issue #79